src/source/RazorEngine.Core.Roslyn/RoslynCompilerServiceBase.cs

```diff
@@ -32,7 +32,7 @@ private class SelectMetadataReference : CompilerReference.ICompilerReferenceVisi
         {
             public MetadataReference Visit(Assembly assembly)
             {
-                return MetadataReference.CreateFromAssembly(assembly);
+                return Visit(assembly.Location);
```

Comment:
Maybe we should get rid from the Assembly-case ("CreateFromAssembly"). It is the an artefact from the hope of early Roslyn builds, which supported an in-memory assembly. But sadly that is no longer the case :(